diracx/values.yaml

```diff
@@ -341,8 +341,10 @@ indigoiam:
 
 ##########################
 
+# ONLY ONE CAN BE ENABLED
+
 mysql:
-  enabled: true
+  enabled: false
```

Comment:
Note: OK for this PR, but a subsequent PR should flip the flag. Separate tests for both backends should come from doubling https://github.com/DIRACGrid/diracx/blob/main/.github/workflows/main.yml#L96